apps/src/templates/progress/ProgressLevelSet.jsx

```diff
@@ -85,6 +85,26 @@ const ProgressLevelSet = React.createClass({
     levels: PropTypes.arrayOf(levelType).isRequired
   },
 
+  getIcon() {
+    const { levels } = this.props;
+    const level = levels[0];
+
+    // TODO - Once we know what locked levels and peer reviews are going to look
+    // like in the redesign, we'll need add logic for those here.
+
+    if (level.icon) {
+      // Eventually I'd like to have dashboard return an icon type. For now, I'm just
+      // going to treat the css class it sends as a type, and map it to an icon name.
```

Comment:
I'm not too sure how strongly I feel about this, but I think dashboard should really be sending down something like `level.iconType = "assessment";` which our view can then turn into `fa-list-ol` in the redesigned dot, and into `null` in the case of the old dot. Rather than sending us classNames, it should be sending us data, which the view can use to determine how to render.